docs/docs/babel.md

```diff
@@ -22,48 +22,25 @@ browsers.
 ## How to use a custom .babelrc file
 
 Gatsby ships with a default .babelrc setup that should work for most sites. If you'd like
-to add custom Babel presets or plugins, we recommend copying our default .babelrc below
-to the root of your site and modifying it per your needs.
+to add custom Babel presets or plugins, you can create your own `.babelrc` at the root of your site, import [`babel-preset-gatsby`](https://github.com/gatsbyjs/gatsby/tree/master/packages/babel-preset-gatsby), and overwrite the `target` option.
+
+```bash
+npm install --save babel-preset-gatsby
+```
 
 ```json5:title=.babelrc
 {
   presets: [
     [
-      "@babel/preset-env",
+      "gatsby",
```

Comment:
Should be `babel-preset-gatsby` instead of `gatsby`. Babel v7 no longer supports shortening `babel-preset-foo` to just `foo` AFAIK.